src/components/editor-menu.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Toolbar, ToolbarGroup, ToolbarTitle } from 'material-ui/Toolbar'
+
+import IconButton from 'material-ui/IconButton'
+import HistoryIcon from 'material-ui/svg-icons/action/history'
+import ArrowDropDown from 'material-ui/svg-icons/navigation/arrow-drop-down'
+
+import darkBaseTheme from 'material-ui/styles/baseThemes/darkBaseTheme'
+import getMuiTheme from 'material-ui/styles/getMuiTheme'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+
+import MainMenu from './menu-component'
+import ViewModeToggleButton from './view-mode-toggle-button'
+
+import Title from './title'
+
+import { prettyDate } from '../notebook-utils'
+
+
+class EditorMenu extends React.Component {
```

Comment:
Since we are breaking these components out into separate files, this seems like a good time to rename this `EditModeToolbar`, since that's more descriptive. I would also rename this file `edit-mode-toolbar.jsx`.